### PR TITLE
OSX build: Remove hardcoded references to Python version

### PR DESCRIPTION
**Pull request by morevnaproject**

### CODE REVIEW

File: 3-package-osx-dmg.sh [hunk 1]
```diff
@@ -199,8 +199,8 @@ mkapp()
 	mkdir -p "${APPCONTENTS}/Frameworks/Python.framework/Versions/${PYTHON_VERSION}/lib/python${PYTHON_VERSION}/"
 	rsync -av --exclude "__pycache__" "${MACPORTS}${PKG_PREFIX}/Frameworks/Python.framework/Versions/${PYTHON_VERSION}/lib/python${PYTHON_VERSION}/" "${APPCONTENTS}/Frameworks/Python.framework/Versions/${PYTHON_VERSION}/lib/python${PYTHON_VERSION}/"
 	pushd "${APPCONTENTS}/Frameworks/Python.framework/Versions/${PYTHON_VERSION}/lib/python${PYTHON_VERSION}/" > /dev/null
-	mkdir -p ../../../../../../lib/python3.6/site-packages
-	ln -sf ../../../../../../lib/python3.6/site-packages site-packages
+	mkdir -p ../../../../../../lib/python${PYTHON_VERSION}/site-packages
```
Comment (ice0) [inline 3-package-osx-dmg.sh]: Why not to use ${APPCONTENTS} here? I think it will be the same:

`${APPCONTENTS}/lib/python${PYTHON_VERSION}`

Comment (morevnaproject) [reply 3-package-osx-dmg.sh]: In next line (`ln -sf ...`) we need to specify relative path (to make sure symlink is correct if AppDir is launched from any location. So, I have used same relative path for `mkdir ...` statement. 

In this case, it is easier to see that paths in both lines are the same. If we will need to fix path, then we will simply fix it to the **same** value in both places. This is simpler and less chances for mistake here. ^__^

Comment (ice0) [reply 3-package-osx-dmg.sh]: Got it.

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**